src/store/modules/getters/staticData.js

```diff
@@ -1,6 +1,12 @@
 import { isWritingNFT } from '~/util/nft';
 
-export const getUserInfoById = state => id => state.userInfos[id];
+export const getUserInfoById = state => id => state.userInfosById[id];
+
```

Comment:
```suggestion
export const userInfosById = state => Object.values(state.userInfosByAddress).reduce(/* code */);

export const getUserInfoById = state => id => state.userInfosById[id];

```

Reply:
Would we still need to maintain id-userInfo map? Since it's going to be removed soon.
By the way, will this modification make the time complexity of `getUserInfoById` become O(N) if we use `reduce` here?

Reply:
for data structure, maybe replace the `userInfosById` with a map with userId as key and address as value is better, so you can leverage the `userInfoByAddressMap`

Reply:
I see, it looks better. But should we further modify `getUserInfoById()` related logic or keep it isolated in this PR?